
From: Ingo Molnar <mingo@elte.hu>

Whitespace cleanups and remove #ifdef to nonexistent option.  No code
impact.

Signed-off-by: Ingo Molnar <mingo@elte.hu>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/include/linux/preempt.h |    6 ------
 25-akpm/kernel/sched.c          |    5 -----
 2 files changed, 11 deletions(-)

diff -puN include/linux/preempt.h~clean-up-preempt-debugging include/linux/preempt.h
--- 25/include/linux/preempt.h~clean-up-preempt-debugging	Thu Oct  7 13:44:54 2004
+++ 25-akpm/include/linux/preempt.h	Thu Oct  7 13:44:54 2004
@@ -52,15 +52,9 @@ do { \
 
 #else
 
-#ifdef CONFIG_PREEMPT_TIMING
-#define preempt_disable()		inc_preempt_count()
-#define preempt_enable_no_resched()	dec_preempt_count()
-#define preempt_enable()		dec_preempt_count()
-#else
 #define preempt_disable()		do { } while (0)
 #define preempt_enable_no_resched()	do { } while (0)
 #define preempt_enable()		do { } while (0)
-#endif
 #define preempt_check_resched()		do { } while (0)
 
 #endif
diff -puN kernel/sched.c~clean-up-preempt-debugging kernel/sched.c
--- 25/kernel/sched.c~clean-up-preempt-debugging	Thu Oct  7 13:44:54 2004
+++ 25-akpm/kernel/sched.c	Thu Oct  7 13:44:54 2004
@@ -2900,9 +2900,6 @@ asmlinkage void __sched preempt_schedule
 	struct task_struct *task = current;
 	int saved_lock_depth;
 #endif
-
-
-
 	/*
 	 * If there is a non-zero preempt_count or interrupts are disabled,
 	 * we do not want to preempt the current task.  Just return..
@@ -3664,8 +3661,6 @@ asmlinkage long sys_sched_yield(void)
 
 static inline void __cond_resched(void)
 {
-
-
 	if (preempt_count() & PREEMPT_ACTIVE)
 		return;
 	do {
_
